tools/sigh.ts

```diff
@@ -518,7 +518,11 @@ function lint(args: string[]): boolean {
     string: ['format']
   });
 
-  const result = saneSpawnSyncWithOutput('git', ['--no-pager', 'grep', '"\\(describe\\.only(\\|it\\.only(\\)"', '*.ts'], {logCmd: true});
+  const result = saneSpawnSyncWithOutput(
+    'grep',
+    ['--include', '\\*.ts', '-R', '"\\(describe\\|it\\)\\.only"', './src'],
```

Comment:
Probably should limit `--include` to `*test.ts`, and at least on linux that flag is glob-based, not regex (so the backslash isn't needed).

Reply:
I think it's possible that someone could include a non `test.ts` file and that would still disable our tests.


The `*` is escaped so that the shell doesn't expand it.

Reply:
Fair enough for the `test` bit. I tried without the `*` escaped (in sigh) and it worked, but it probably doesn't matter much.